### PR TITLE
run cron jobs in docker

### PR DESCRIPTION
**Pull request by tnextday**
#77

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -94,9 +95,12 @@ COPY supervisor.conf /etc/supervisor/conf.d/
 
 # Entrypoint
 ADD start /app/bin/
-RUN chmod a+rx /app/bin/start
+COPY crontab.txt /app/crontab.txt
+RUN chmod a+rx /app/bin/start \
+    && crontab -u weblate /app/crontab.txt
 
 ENV DJANGO_SETTINGS_MODULE weblate.settings
+ENV WEBLATE_OFFLOAD_INDEXING 1
```
Comment (nijel) [inline Dockerfile]: This should be rather completely removed and hardcoded in the settings.py instead.

File: Dockerfile [hunk 2]
```diff
@@ -94,9 +95,12 @@ COPY supervisor.conf /etc/supervisor/conf.d/
 
 # Entrypoint
 ADD start /app/bin/
-RUN chmod a+rx /app/bin/start
+COPY crontab.txt /app/crontab.txt
```
Comment (nijel) [inline Dockerfile]: As the file is no needed after installation, I'd rather copy it to /tmp what can be done together with `requirements.txt` few lines above.

Comment (tnextday) [reply Dockerfile]: fixed

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**